assets/constants.json

```diff
@@ -7,5 +7,6 @@
     "Mathematics", "Medicine", "Music", "Philosophy", "Physics", "Poetry",
     "Probability", "Programming", "Puzzles", "Reading", "Spanish", "Sport",
     "Statistics", "Trigonometry", "Welcome"],
-  "DISABLED_EXPLORATION_IDS": ["5"]
+  "DISABLED_EXPLORATION_IDS": ["5"],
+  "NEW_STATE_TEMPLATE":{"content":[{"type":"text","value":""}],"interaction":{"answer_groups":[],"confirmed_unclassified_answers":[],"customization_args":{"rows":{"value":1},"placeholder":{"value":"Type your answer here."}},"default_outcome":{"dest":"(untitled state)","feedback":[],"param_changes":[]},"fallbacks":[],"id":"TextInput"},"param_changes":[]}
```

Comment:
OK, so, I looked up NULL_INTERACTION_DICT and it seems like it is not used elsewhere. (I'm not sure why your A transitioned from NULL_INTERACTION_DICT to DEFAULT_INIT_STATE_NAME.)

But since that's the case I think you can get rid of A and move B to constants.json, similar to @vojtechjelinek's PR earlier. Then, when you need B in the frontend later, you'd get it from constants.json in the same way. Let me know if I'm missing something here.